### PR TITLE
proto changes for vlan id field in app virtual interface configuration

### PR DESCRIPTION
**Pull request by gkodali-zededa**
Will commit the built API files after review and before final merge.

Signed-off-by: gkodali-zededa <gkodali@zededa.com>

### CODE REVIEW

File: api/proto/config/netconfig.proto [hunk 1]
```diff
@@ -48,6 +48,11 @@ message NetworkAdapter {
 
   // firewall
   repeated ACE acls = 40;
+
+  // access port vlan id
+  // app interface with access vlan id of zero will be treated as trunk port
+  // valid vlan id range: 1 - 4094
```
Comment (eriknordmark) [inline api/proto/config/netconfig.proto:54]: Nit: max is 4093.
In addition some documentation claims 1 and 4093 is reserved. But I don't think we need to care about that.

Comment (gkodali-zededa) [reply api/proto/config/netconfig.proto:54]: will change the comment

Comment (gkodali-zededa) [reply api/proto/config/netconfig.proto]: DOne

**Review by eriknordmark — APPROVED** [review]
LGTM

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**